Automattic/wp-calypso
# Checkout: Remove SecurePaymentForm

#### Changes proposed in this Pull Request

This removes the `SecurePaymentForm` component from "old" checkout, which is no longer in use and many of its otherwise unused dependencies. This will allow us to eventually remove the `TransactionFlow` object which is the last major consumer of the deprecated `CartStore`.

#### Testing instructions

Verify that these components and functions are not being used anywhere. Just compiling Calypso should be sufficient for the files that were fully removed. 

The single functions that were removed should be verified as unused individually:

- isPaidForFullyInCredits
- isAntiSpamConflicting
- isProductConflictingWithSite
- getCheckoutIncompatibleProducts

And the type:

- IncompatibleProducts
